### ISSUE TITLE
[Accessability Bug] On Hold (waiting for Ant Design) Inline Error message is not read by screen reader

### ISSUE THREAD

**Issue by AminaSalah** [CDH-Studio/UpSkill#1548]
**Describe the bug**
Contact Information Form:
Inline Error message is not read by screen reader. The error should be read to screen reader users on focus. At minimum its recommended to say "invalid" when tabbing to a field with invalid input. 

- "Invalid" technically passes, but it would be more helpful to a non-sighted user if the inline error can be read out as well
- OR
- The error would be read out on submit of the form and be included in the submission error message (currently it is not, see issue #1547)

**To Reproduce**

1. Login
2. Accept the privacy agreement
3. Navigate to the Contact Information Form
4. Tab through the form

**Comment by dynamic11** [comment]
@Rizvida1 is this related to what you are working on now?

**Comment by Rizvida1** [comment]
@dynamic11 Yes this is what I'm currently working on

**Comment by MenaMachado** [comment]
Put PR number from Ant Design

**Comment by MenaMachado** [comment]
Waiting for Ant Design to fix issue.

**Comment by Rizvida1** [comment]
Waiting for https://github.com/ant-design/ant-design/pull/31104
